public/assets/css/home.css

```diff
@@ -1,9 +1,3 @@
-nav {
-    padding: 1em 1em 0em 1em;
-    display: flex;
-    justify-content: space-between ;
-}
-
 .questionmark-button, .music-button {
```

Comment:
Hi, .questionmark-button and .music-button doesn't exist anymore in the view. You should consider to delete it

Reply:
changes done